lib/navigation/preprocess_imu.hpp

```diff
@@ -35,18 +35,17 @@ class ImuPreprocessor {
   }
 
   template<std::size_t N>
-  Quartiles getQuartiles(const std::array<core::Float, N> &imu_data)
+  Quartiles getQuartiles(const std::array<core::Float, N> &accelerometer_data)
   {
     std::array<core::Float, N> accelerometer_data_copy;
-    std::copy(imu_data.begin(), imu_data.end(), accelerometer_data_copy.begin());
+    std::copy(
+      accelerometer_data.begin(), accelerometer_data.end(), accelerometer_data_copy.begin());
 
     std::sort(accelerometer_data_copy.begin(), accelerometer_data_copy.end());
-
     Quartiles quartiles;
-    quartiles.q1     = getSpecificQuartile(accelerometer_data_copy, 0.25);
-    quartiles.median = getSpecificQuartile(accelerometer_data_copy, 0.5);
-    quartiles.q3     = getSpecificQuartile(accelerometer_data_copy, 0.75);
-    return quartiles;
+    return {quartiles.q1     = getSpecificQuantile(accelerometer_data_copy, 0.25),
+            quartiles.median = getSpecificQuantile(accelerometer_data_copy, 0.5),
+            quartiles.q3     = getSpecificQuantile(accelerometer_data_copy, 0.75)};
```

Comment:
it should be possible to drop the `Quartiles quartiles` bit and then leave out `quartiles` in the return statement.

Reply:
https://pdimov.github.io/blog/2020/09/07/named-parameters-in-c20/